docs/StrategyFormatChangelog.md

```diff
@@ -1,5 +1,18 @@
 # Změny formátu strategie
 
+## Robozonky 5.6.0
```

Comment:
User facing doc describing the change: https://github.com/RoboZonky/natural-strategy-setup/blob/d589004219cebfd862a4d8e8e5a12bc65563e2cb/docs/StrategyFormatChangelog.md